cloud/scope/powervs_machine_test.go

```diff
@@ -111,83 +122,1003 @@ func newPowerVSInstance(name, networkID, mac string) *models.PVMInstance {
 	}
 }
 
-func newDHCPServer(serverID, networkID string) models.DHCPServers {
-	return models.DHCPServers{
-		&models.DHCPServer{
-			ID: ptr.To(serverID),
-			Network: &models.DHCPServerNetwork{
-				ID: ptr.To(networkID),
+func newDHCPServer(serverID, networkID string) models.DHCPServers {
+	return models.DHCPServers{
+		&models.DHCPServer{
+			ID: ptr.To(serverID),
+			Network: &models.DHCPServerNetwork{
+				ID: ptr.To(networkID),
+			},
+		},
+	}
+}
+
+func newDHCPServerDetails(serverID, leaseIP, instanceMac string) *models.DHCPServerDetail {
```

Comment:
these functions can be moved to common_test.go file.

Reply:
moved